conf/messages.en

```diff
@@ -93,6 +93,7 @@ journeys.capital-allowances-annual-investment-allowance = Annual Investment Allo
 journeys.capital-allowances-writing-down-allowance = ‘Writing down’ allowance
 journeys.capital-allowances-special-tax-sites = Freeport sites
 journeys.capital-allowances-structures-buildings = Structures and Buildings Allowance
+journeys.adjustments-work-out-profit-and-loss = Work out your adjusted profit or loss
```

Comment:
This is the text for Journey.ProfitOrLoss so should be journey.profit-or-loss

Reply:
Done :)